pypit/arsave.py

```diff
@@ -428,8 +428,16 @@ def save_1d_spectra_fits(slf, standard=False, clobber=True, outfile=None):
     Returns
     -------
     """
-    # Primary header
+    # Primary hdu
     prihdu = pyfits.PrimaryHDU()
+    # Add critical data to header
+    idx = slf._idx_sci[0]
+    prihdu.header['RA'] = fitsdict['ra'][idx]
+    prihdu.header['DEC'] = fitsdict['dec'][idx]
+    prihdu.header['EXPTIME'] = fitsdict['exptime'][idx]
+    prihdu.header['MJD-OBS'] = fitsdict['time'][idx]
+    prihdu.header['DATE'] = fitsdict['date'][idx]
+    # Begin hdu list
```

Comment:
If you don't mind me suggesting a few extras here...

* Object Name
* Latitude/Longitude/Altitude of observatory
* Longitude of observatory
* velocity correction in km/s (as well as the type of correction, i.e. barycentric, heliocentric etc.)
* number of spectra
* number of pixels in spectral direction

... that's all I can think of reasonably for now.